src/test/kotlin/com/tradeshift/blayze/ModelTest.kt

```diff
@@ -25,14 +26,14 @@ class ModelTest {
             ).batchUpdate(listOf(
                     "p" to Counter(mapOf("awesome" to 7, "terrible" to 3, "ok" to 19)),
                     "n" to Counter(mapOf("awesome" to 2, "terrible" to 13, "ok" to 21))
-            ))),
+            ), null)),
```

Comment:
ugh, I really dislike having to pass around these null values everywhere. Is there something we can do? Maybe set the default null argument in the interface?